projects/dex-ui/src/components/Frame/Footer.tsx

```diff
@@ -22,6 +30,9 @@ export const Footer = () => (
     <SmallBox href="https://github.com/BeanstalkFarms/Basin" rel="noopener noreferrer" target="_blank">
       <Github width={20} />
     </SmallBox>
+    <SmallBox href="https://app.bean.money" rel="noopener noreferrer" target="_blank">
```

Comment:
let's change this link to `https://bean.money` @Bean-Sama

Reply:
sounds good!